syft/generic/pointers/pointer_tensor.py

```diff
@@ -418,9 +432,16 @@ def value(self, *args, **kwargs):
         Returns:
             A pointer to a Tensor
         """
-        command = ("value", self, args, kwargs)
+        ptr_id = self.id_at_location
+        source = self.owner.id
+        _kwargs = {}
+        _args = ()
+        message = TensorCommandMessage.communication(
```

Comment:
I'm not sure if this one should be a computation or a communication. 🤔 

The more I think about it the more I think you have it right, but I don't really get what `value` is for since we also have `get`. 🤷

Reply:
(got ride of it : )